src/main/webapp/components/BaseFormView.jsx

```diff
@@ -535,6 +535,10 @@ class BaseFormView extends PureComponent {
     };
 
     saveData = () => {
+        if (this.isOAuth) {
```

Comment:
Can we have it similarly to where we are already removing `name` from `body` below.

Reply:
Done